### PR TITLE
fix/disappearing-navbar

### PR DESCRIPTION
**Pull request by sameer-soni**
## Linked Issue
fixed: #823

## Description
The problem was that the first page has zindex of 1034 higher than the navbar, so when user was in first page, navbar was not showing:
bug

so what i did is, I made the navbar z-index = 2000. I did in .navbar class. Now, it's fixed.
fixe

## Code of Conduct

> By submitting this pull request, you agree to follow our [Code of Conduct](https://virtualcoffee.io/code-of-conduct/)

### CODE REVIEW

**Review by danieltott — CHANGES_REQUESTED** [review]
Hey @sameer-soni - thanks for the PR!

The issue here isn't that it's hidden on the home page - that is the expected behavior.

The problem is that when the page overscrolls, the nav-bar becomes visible.

Check out [the video](https://github.com/Virtual-Coffee/virtualcoffee.io/issues/823#issue-1608425310) and [my suggestion](https://github.com/Virtual-Coffee/virtualcoffee.io/issues/823#issuecomment-1478343758) in the original issue.

To sum up:
- Nav bar should be initially hidden on the home page
- Nav bar should be visible if it has focus
- Nav bar should scroll into view on home page
- No pages should overscroll past the top

**Comment by danieltott** [comment]
@sameer-soni Are you interested in fixing this up?
